hw/top_earlgrey/sw/autogen/top_earlgrey.h

```diff
@@ -166,6 +166,22 @@
  */
 #define TOP_EARLGREY_ALERT_HANDLER_BASE_ADDR 0x40130000u
 
+/**
```

Comment:
is this addition a rebase error somewhere? in the past, or in this PR?

Reply:
re-generated the software code based on the tip of the master and it creates this.

Reply:
This must have been my fault when solving a merge conflict.